js/pages/cohort-definitions/cohort-definitions.js

```diff
@@ -44,7 +46,7 @@ define([
 		
 			this.isAuthenticated 	= authApi.isAuthenticated;
 			this.canReadCohorts 	= ko.pureComputed(() => (config.userAuthenticationEnabled && this.isAuthenticated() && authApi.isPermittedReadCohorts()) || !config.userAuthenticationEnabled);
-			this.canCreateCohort 	= ko.pureComputed(() => (config.userAuthenticationEnabled && this.isAuthenticated() && authApi.isPermittedCreateCohort()) || !config.userAuthenticationEnabled);			
+			this.canCreateCohort 	= ko.pureComputed(() => (sharedState.vocabularyUrl() && (config.userAuthenticationEnabled && this.isAuthenticated() && authApi.isPermittedCreateCohort()) || !config.userAuthenticationEnabled));
```

Comment:
We agreed not to disable cohort creation when no datasources is available.
That was why ossue was closed.
So this changes is unnecessary.